.env-devel

```diff
@@ -141,6 +141,7 @@ DYNAMIC_SCHEDULER_UI_STORAGE_SECRET=adminadmin
 FUNCTION_SERVICES_AUTHORS='{"UN": {"name": "Unknown", "email": "unknown@osparc.io", "affiliation": "unknown"}}'
 
 WEBSERVER_LICENSES={}
+WEBSERVER_FOGBUGZ={}
```

Comment:
this enables it by default. is that what we want here? or is it supposed to be optional?